plugins/modules/zos_copy.py

```diff
@@ -563,6 +563,61 @@
     returned: success
     type: str
     sample: SAMPLE.SEQ.DATA.SET
+dest_created:
+    description: Indicates whether the module created the destination.
+    returned: success and if dest was created by the module.
+    type: bool
+    sample: true
+destination_attributes:
+    description: Attributes of a dest create by the module.
```

Comment:
Type here: "create" instead of "created". Just pointing it out because this will be part of the collection docs.

Reply:
I guess I miss some created in the last change request sorry.

Reply:
I saw that this part of the documentation was already correct and then you introduced the typo over the last commits. It just a matter of reverting that change. Sorry for hammering on this point, I'm just insisting because this will be part of the documentation for the collection, everything else is pretty good and ready to be merged into `dev`.

Reply:
Ok, I think is done